packages/core-flows/src/invite/steps/delete-invites.ts

```diff
@@ -0,0 +1,28 @@
+import { ModuleRegistrationName } from "@medusajs/modules-sdk"
+import { IUserModuleService } from "@medusajs/types"
+import { StepResponse, createStep } from "@medusajs/workflows-sdk"
+
+export const deleteInvitesStepId = "delete-invites-step"
+export const deleteInvitesStep = createStep(
+  deleteInvitesStepId,
+  async (input: string[], { container }) => {
+    const service: IUserModuleService = container.resolve(
+      ModuleRegistrationName.USER
+    )
+
+    await service.softDeleteInvites(input)
```

Comment:
todo: I would suggest to be careful here, just to be sure, but if you pass an empty array it will soft delete all the invites table. So maybe like the other one we should check that there is length before

Reply:
To me, sounds like we should have this check further down the stack 😅

Reply:
yes true done https://github.com/medusajs/medusa/pull/6396